lib/utils.js

```diff
@@ -147,6 +147,33 @@ exports.isSnakeCased = function(name) {
 };
 
 /**
+ * Returns the function expression node if the provided node is an iffe,
+ * other returns undefined.
+ *
+ * @param  {Object} node
+ * @return {Object|undefined}
```

Comment:
@mikesherov Absolutely valid. http://usejsdoc.org/tags-type.html It's called nullable type.